srcs/app/http/response/HttResponseFactory.cpp

```diff
@@ -0,0 +1,66 @@
+#include <HttpResponseFactory.hpp>
```

Comment:
This filename is missing the 'p' after the Htt.

HttResponseFactory.cpp -> HttpResponseFactory.cpp

Reply:
Good catch! I renamed several files and this could be due to it. Thanks!